pilz_control/test/integrationtest_pilz_joint_trajectory_controller.py

```diff
@@ -182,6 +185,20 @@ def wait_for_end_of_stop_observation(self):
             return self._stop_trajectory_ok
 
 
+    def _try_to_get_actual_position(self):
+        with self._controller_state_lock:
+            return self._actual_position
```

Comment:
Not really part of this PR, but why is this variable called `_actual_position`? Considering the context it seems that `_position` would do the trick.